protocol/messenger_local_backup.go

```diff
@@ -71,15 +71,15 @@ func (m *Messenger) BackupDataLocally(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
-	// _, settings, errors := m.prepareSyncSettingsMessages(clock, true)
-	// if len(errors) != 0 {
-	// 	// return just the first error, the others have been logged
-	// 	return errors[0]
-	// }
-	// woAccountsToBackup, err := m.backupWatchOnlyAccounts()
-	// if err != nil {
-	// 	return 0, err
-	// }
+	_, settings, errors := m.prepareSyncSettingsMessages(clock, true)
+	if len(errors) != 0 {
+		// return just the first error, the others have been logged
+		return errors[0]
```

Comment:
```suggestion
	_, settings, errs := m.prepareSyncSettingsMessages(clock, true)
	if len(errors) != 0 {
		return errors.Join(errs)
```

or even better to join the errors inside `prepareSyncSettingsMessages`